package.json

```diff
@@ -72,23 +72,23 @@
         "@material-ui/core": "^4.7.2",
         "@material-ui/icons": "^4.5.1",
         "@material-ui/lab": "^4.0.0-alpha.35",
-        "@storybook/addon-actions": "^5.2.8",
+        "@storybook/addon-actions": "^5.3.19",
         "@storybook/addon-console": "^1.2.1",
-        "@storybook/addon-knobs": "^5.2.8",
-        "@storybook/addon-links": "^5.2.8",
-        "@storybook/addon-storysource": "^5.2.8",
-        "@storybook/addons": "^5.2.8",
-        "@storybook/react": "^5.2.8",
-        "@storybook/storybook-deployer": "^2.8.1",
+        "@storybook/addon-knobs": "^5.3.19",
+        "@storybook/addon-links": "^5.3.19",
+        "@storybook/addon-storysource": "^5.3.19",
+        "@storybook/addons": "^5.3.19",
+        "@storybook/react": "^5.3.19",
+        "@storybook/storybook-deployer": "^2.8.6",
         "babel-loader": "^8.0.6",
         "css-loader": "^2.1.1",
         "formik": "^1.5.8",
         "husky": "^1.3.1",
         "jest-junit-reporter": "^1.1.0",
-        "prettier": "1.16.4",
+        "prettier": "2.0.5",
         "pretty-quick": "^1.11.1",
         "prop-types": "^15.7.2",
-        "react-intl": "^2.9.0",
+        "react-intl": "^5.0.2",
```

Comment:
We're only on `3.12.0` in sonora, so this one makes me a little nervous since it's 2 major versions higher.  Did everything look ok?  Not sure how much we're really using `react-intl` in this repo, so not sure if any major issues would be easy to find.

Reply:
Storybook loaded fine. I will see if I can check `Sonora` with 5.x locally

Reply:
Hmm nope.  Latest version `react-intl` doesn't have `FormattedHTMLMessage` anymore 😖. I will roll this back to existing version. 
Their API has changed a lot. That and having to pass `intl` everywhere,  am thinking we should look elsewhere for i18N.

Reply:
Actually I will update it match what we have in Sonora - `^3.12.0`

Reply:
Agreed - an alternative library would be nice :thumbsup:

Reply:
@ash3rz - is the PR good to go ?